docs/package_list.md

```diff
@@ -43,6 +39,9 @@
 | agent/valory/registration_start_up/0.1.0                      | `bafybeibpnjhetlif6zesescxcypqcscedb7u3dgpmv4nfbbwabuwj42k3m` |
 | agent/valory/simple_abci/0.1.0                                | `bafybeiez76fbbul4xyizbytr424onz3s2dd2ct3lyjip5wkjylk7opmxmi` |
 | agent/valory/test_abci/0.1.0                                  | `bafybeifllqlz32rz3uglgvug3vffr753stagotmijzu2gws24z2qmxg6ce` |
+| contract/valory/multisend/0.1.0                               | `bafybeig45676hbh4c3p3mujrrskxgxww4cxdyyginlg5rmmav6orv4gtya` |
+| contract/valory/uniswap_v2_erc20/0.1.0                        | `bafybeibkiwytuhhvxbisoxcybrx4lfstk6bvtriltles7fxxpy37yq45ja` |
+| contract/valory/uniswap_v2_router_02/0.1.0                    | `bafybeigk46llosgsfz3zp7gdz44t4libzqqqq7zi6d5wsjpz43evn6ww4m` |
```

Comment:
These two (uniswap_v2_erc20 and uniswap_v2_router_02) should be moved too? I think they're only used in the liquidity rebalancing side?

We can leave `multisend` here as that contract will be needed in many apps.

Reply:
Well, I noticed that comment *"...and Uniswap contracts, ..."*:

https://github.com/valory-xyz/open-autonomy/blob/463bb550e7aeb68b8f5af7afdcb6b3ac846a16cc/plugins/aea-test-autonomy/aea_test_autonomy/docker/amm_net.py#L49

Reply:
Ah ok. Yes but these are not the packages. Although the code matches